lib/constants.js

```diff
@@ -197,6 +197,13 @@ module.exports = Object.freeze({
       DESC: -1
     }
   },
+  RESOURCE_STATE: {
+    IN_QUEUE: 'in queue',
+    IN_PROGRESS: 'in progress',
+    DEPLOYED: 'deployed',
```

Comment:
Deployed may not be necessary. It was previously added to handle the security group scenario. Now that it is handled by platform managers, it won’t be needed.